client/src/translations/en.json

```diff
@@ -255,6 +255,7 @@
   "pdf": "PDF",
   "pending": "Pending",
   "percentage_stack_bars": "Percentage Stack Bars",
+  "pick_a_default_value": "Pick a default value",
```

Comment:
When adding a new translation key, we need to take care of the `es.json` and `fr.json` files too

Reply:
What're the translations for Spanish and french?

Reply:
I have started to add keys: use Google translate to get something (that hopefully will trigger a native speaker to react) and then I assign someone that knows the languages as reviewers. Not sure they love this practice but.. seems to work :-)